.github/workflows/windows.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - name: build library
       shell: bash
       run: |
-        cmake --build build  --target install -- -m
+        cmake --build build --parallel 2 --target install -- -m
```

Comment:
That's already what `-m` is doing.

Reply:
Okay: https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-command-line-reference?view=vs-2022#switches

Which way is nicer? Shall we let `cmake` impose this, or shall we pass the flag to the native tool as we do now?

Personally, I find the `cmake` way easier to read.

Reply:
I like `--parallel 2` better.